<launch>
  <arg name="model" default="$(find lophorina)/urdf/lophorina3.urdf"/>
  <arg name="rvizconfig" default="$(find lophorina)/rviz/urdf.rviz" />

  <include file="$(find lophorina)/launch/gazebo.launch">
    <arg name="model" value="$(arg model)" />
  </include>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" />

  <rosparam command="load"
            file="$(find lophorina)/config/joints.yaml"
            ns="lophorina_joint_state_controller" />
  <rosparam command="load"
            file="$(find lophorina)/config/tilt.yaml"
            ns="lophorina_tilt_controller" />  
  <rosparam command="load"
            file="$(find lophorina)/config/propellers.yaml"
            ns="lophorina_prop_controller" />  

  <node name="lophorina_controller_spawner" pkg="controller_manager" type="spawner"
    args="lophorina_joint_state_controller
          lophorina_tilt_controller
          lophorina_prop_controller"/>

</launch>
